defos/src/defos_html5.cpp

```diff
@@ -144,6 +144,36 @@ void defos_set_window_title(const char* title_lua) {
     EM_ASM_({document.title = UTF8ToString($0)}, title_lua);
 }
 
+void defos_set_window_icon(const char *icon_path)
+{
+    EM_ASM_({
+        document.head || (document.head = document.getElementsByTagName('head')[0]);
+        function changeFavicon(src) {
+            var link = document.createElement('link');
+            var oldLink = document.getElementById('dynamic-favicon');
```

Comment:
If the user has a `<link rel='shortcut icon'>` already in the document that wasn't inserted by defos, he'll end up having two `<link>`'s. Maybe do `var oldLink = document.querySelector('link[rel="shortcut icon"]')`?

Reply:
done